packages/core-components/src/layout/SignInPage/providers.tsx

```diff
@@ -88,6 +93,22 @@ export const useSignInProviders = (
   const apiHolder = useApiHolder();
   const [loading, setLoading] = useState(true);
 
+  const { t } = useTranslationRef(coreComponentsTranslationRef);
+  // User was redirected back to sign in page with error from auth redirect flow
+  const [searchParams, _setSearchParams] = useSearchParams();
+
+  useMountEffect(() => {
+    const errorParam = searchParams.get('error');
+    if (errorParam) {
+      errorApi.post(
+        new ForwardedError(t('signIn.loginFailed'), {
+          name: 'Error',
```

Comment:
Could you make the second arg a regular error object? `new Error(decodeURIComponent(errorParam))`